build.sbt

```diff
@@ -212,17 +212,14 @@ lazy val buildSettings = Seq(
 
 lazy val commonSettings = Seq(
   scalacOptions ++= commonScalacOptions.diff(Seq(
-    "-Xfatal-warnings", 
+    "-Xfatal-warnings",
     "-language:existentials",
     "-Ywarn-dead-code",
     "-Ywarn-numeric-widen",
     "-Ywarn-value-discard"
   )),
-  resolvers ++= Seq(
-    "bintray/non" at "http://dl.bintray.com/non/maven",
-    Resolver.sonatypeRepo("snapshots")
-  ),
-  libraryDependencies += "org.typelevel" %%% "machinist" % "0.4.1"
+  libraryDependencies += "org.typelevel" %%% "machinist" % "0.5.0",
+  libraryDependencies += "org.typelevel" %%% "algebra" % "0.5.2-SNAPSHOT"
```

Comment:
I see this is a SNAPSHOT - are there any pre-requisites on algebra before merging this (aside from a release)?